src/metabase/query_processor/middleware/annotate.clj

```diff
@@ -159,6 +159,10 @@
            (select-keys (infer-expression-type expression) type-info-columns)))
        clauses))
 
+    (mbql.u/is-clause? :convert-timezone expression)
+    {:converted_timezone (nth expression 2)
+     :base_type          :type/DateTimeWithTZ}
```

Comment:
Shouldn't this be `:type/DateTime`?

Reply:
I changed it here https://github.com/metabase/metabase/pull/25698/commits/f92ddafd562f6670fe82c41378991324299426da